packages/compass-aggregations/src/components/focus-mode/focus-mode-modal-header.tsx

```diff
@@ -32,6 +36,7 @@ type FocusModeModalHeaderProps = {
   stageIndex: number;
   isEnabled: boolean;
   stages: Stage[];
+  insight: Signal | undefined;
```

Comment:
```suggestion
  insight?: Signal | undefined;
```

Reply:
I just addressed this in [9259570](https://github.com/mongodb-js/compass/pull/4564/commits/92595706a65bedb1eb68c852e0c0ae337330acea)